documentation-site/pages/theming/custom-themes.mdx

```diff
@@ -57,19 +57,19 @@ export default function App() {
 ```
 
 If you'd like to use a different theme in one part of your application, \
-you can locally wrap that part with the ThemeProvider to override the theme locally:
```

Comment:
In this part of the docs the `ThemeProvider` usage would be the correct recommendation.

Reply:
would it work with the BaseProvider too? if so, I think we should let folks know only one way to do it, so it won't be confusing